app/mailers/rent_mailer.rb

```diff
@@ -0,0 +1,13 @@
+class RentMailer < ApplicationMailer
+  # Subject can be set in your I18n file at config/locales/en.yml
+  # with the following lookup:
+  #
+  #   en.rent_mailer.rent_created.subject
+  #
+  def rent_created
+    @rent = Rent.last
```

Comment:
Try receiving the id of the rent as param, if not, you are going to affront a race condition here, if you receive the id as parameter, you can just find the rent, like a controller show endpoint